Dockerfile

```diff
@@ -4,14 +4,12 @@ FROM python:slim AS wheelstage
 RUN pip install build
 RUN apt-get update && \
     apt-get install -y --no-install-recommends git
-COPY . /src/qsiprep
-RUN python -m build /src/qsiprep
 
 FROM pennbbl/qsiprep_build:24.4.29
 
-# Install qsiprep wheel
-COPY --from=wheelstage /src/qsiprep/dist/*.whl .
-RUN pip install --no-cache-dir $( ls *.whl )
+# Install qsiprep
```

Comment:
I'm not sure. I needed to be able to overwrite the source code when mounting the local version, but the old approach didn't seem to have a copy in the Docker image.

Reply:
It's been a little while since I made that change, so my memory is a little fuzzy, but I believe I tried the regular mount point (`/usr/local/miniconda/lib/python3.10/site-packages/qsiprep`) and it didn't work with the old way. Def worth revisiting in case I made a mistake there.

Reply:
Oh wait, I think I remember what the issue was! I didn't know how to install with the test dependencies using the old way.